docs/source/usage/cli.rst

```diff
@@ -180,13 +180,13 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                                               |
-+----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --candidate <node id> --ssh-public-key <path-to-key>`` |
```

Comment:
Do we want to list  resource-class options in the doc? So that users know the exact type of the resource-class they should input.

Reply:
I don't think so, at least not here; maybe in MOC ESI specific documentation, but I don't think there's a strong need to (especially since there are new types of machines being added occasionally).